fastlane/Fastfile

```diff
@@ -0,0 +1,52 @@
+default_platform(:ios)
+
+platform :ios do
+  desc "Build and test the Pod"
+  lane :build_and_test do
+    run_tests(
+      scheme: "wpxmlrpc-iOS"
+    )
+  end
+
+  desc "Validate Podspec"
+  lane :validate_podspec do
+    pod_lib_lint(
+      use_bundle_exec: true,
+      verbose: true,
+    )
+  end
+
+  desc "Publish Podspec"
+  lane :publish_podspec do
+    require 'cocoapods-core'
+
+    pod_push(
+      use_bundle_exec: true,
+      path: "wpxmlrpc.podspec",
+      verbose: true,
+    )
+
+    podspec = Pod::Spec::from_file('../wpxmlrpc.podspec')
+
+    slack(
+      message: ":tada: #{podspec.module_name} #{podspec.version} successfully published to CocoaPods trunk!\nIt will take a few minutes for this version to be deployed to the CocoaPods CDN.",
+      success: true,
+      slack_url: ENV['PODS_SLACK_WEBHOOK'],
+      payload: [
+          "Build URL" => ENV["CIRCLE_BUILD_URL"],
+       ]
+    )
+  end
```

Comment:
The orb already has all the logic for this, so I wonder what's the benefit of replicating this as a lane? 🤔